nemo/collections/common/tokenizers/en_ja_tokenizers.py

```diff
@@ -43,22 +38,13 @@ def detokenize(self, tokens: List[str]) -> str:
         Returns:
             detokenized Japanese or English string
         """
-        text = self.sp_tokenizer.ids_to_text([int(t) for t in tokens])
-        # We need to explictly specify splitting on spaces to avoid
-        # detokenizing a sequence of characters.
-        return self.moses_detokenizer.detokenize(text.split(' '))
-
-    def sp_tokenize(self, text: str) -> str:
-        return ' '.join(self.sp_tokenizer.text_to_ids(text))
+        return self.moses_detokenizer.detokenize(tokens)
 
     def tokenize(self, text):
         """
         Tokenizes text using Moses -> Sentencepiece.
         """
-        if self.sp_tokenizer is None:
-            raise ValueError("Need valid sp_tokenizer_model_path, found None")
-        text = self.moses_tokenizer.tokenize(text, escape=False, return_str=True)
-        return self.sp_tokenize(text)
+        return self.moses_tokenizer.tokenize(text)
```

Comment:
Do you need to set `escape=False` here?

Reply:
i think escaping is fine for en<>ja. the preprocessing pipeline escapes, do we have to unescape. and i checked to make sure it is undone correctly. but are you concerned about this behavior in general?